packages/office-ui-fabric-react/src/components/Callout/Callout.test.tsx

```diff
@@ -133,4 +133,44 @@ describe('Callout', () => {
         expect(threwException).to.be.false;
     });
 
+    it('Callout passes event to onDismiss prop', () => {
```

Comment:
minor note; with mocha tests you "describe" something by saying what "it" can do.

```ts
describe('Jon', () => {
  it('can code', () => {});
  it('can do stunts', () => {});
});
```
Try to follow that convention rather than saying `it('subject does things')` which doesn't read quite the same.